assets/js/module/iframePreview.js

```diff
@@ -63,4 +64,16 @@ export default class IframePreview {
             self.loadIframes(iframes)
         });
     }
+
+    #changeSelfTargetLinksToParent(body) {
```

Comment:
Why a #? Does that mean something specific?

Reply:
It specify a private method. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

https://caniuse.com/mdn-javascript_classes_private_class_methods